spec.html

```diff
@@ -44450,6 +44435,22 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-issamebuffer" type="abstract operation">
+        <h1>
+          IsSameBuffer (
```

Comment:
Maybe instead call this something like `IsBackedBySameBuffer`?

Reply:
I guess I was thinking `IsBackedBySameData`. "Buffer" to me was data block, not ArrayBuffer.